src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -2820,6 +2820,120 @@ class TezosDatabaseOperationsTest
       )
     }
 
+    "should aggregate with datePart aggregation" in {
+      val oneDay = 3600 * 24 * 1000
+      val feesTmp = List(
+        FeesRow(0, 2, 4, new Timestamp(0), "kind"),
```

Comment:
Just for the record. Usually using `java.time.Instant` is better than using old and mutable structures from `java.sql`. In that case, you could use `.plusSeconds(1)`. I'm not saying that we should migrate immediately but we should definitely keep that in mind.

Reply:
I agree that we should use instants. But it is harder than it looks. We use generated slick tables which are using `Timestamp`. In this case there is no way to use `Instant`.

Reply:
@bhaskarkishore: ping